Makefile

```diff
@@ -2,19 +2,30 @@ service := secureopenbanking-uk-iam-initializer
 gcr-repo := sbat-gcr-develop
 binary-name := initialize
 
+define clone_config
+	if [ -d "config-repo" ]; then rm config-repo -rf; fi
+	if [ -d "config" ]; then rm config -rf; fi
+	git clone git@github.com:SecureApiGateway/fr-platform-config.git config-repo
+	mkdir config
+	mv config-repo/config/* config
+	rm -rf config-repo
+endef
```

Comment:
I think this should be a target, then build and test-ci can depend on the target

Reply:
I've done this now and added the target for build, but not for test-ci as that's what the github action calls and we'll already have the config repo cloned